cartridges/int_cloudinary/cartridge/controllers/Cloudinary.js

```diff
@@ -19,6 +19,8 @@ Cloudinary.GetContentImage = function () {
     var pageType = params.pageType.stringValue;
     var isUrl = params.isUrl.stringValue ? JSON.parse(params.isUrl.stringValue) : false;
     var contentAttributes = params.attributes ? params.attributes.stringValue : '';
+    var removeQuotes = contentAttributes.slice(-1) === '"' ? contentAttributes.substring(0, contentAttributes.length -1) : contentAttributes;
```

Comment:
@asad-rafter I'm not sure about this double-quotes manipulation. I expected the customAttriibutes to be a regular string like the URL and the altTest. Please explain/revise

Reply:
@yuval-cloudinary, This is for additional attributes, the user can add as many as they want if we go with approach like URL and altText, its static, and as you see for URL and altText there is relevant code for them. Let me know if that helps.

Reply:
@asad-rafter When using the same approach as URL and altText, additional properties would look like this:
'contentAttributes','prop1="val1" prop2="val2" prop3="val3"'
The advantage is a simple and clear syntax for users as well as in the code

Reply:
@yuval-cloudinary again we need to remove quotes from start and end otherwise it will consider it as single value for 'contentAttributes'. This is default behavior of SFCC. We faced this in our status meeting if you remember, then we removed quotes and it started working, for better user experience we can add quotes in content asset but we have to remove them before rendering.